recipe/meta.yaml

```
@@ -19,28 +18,30 @@ requirements:
   host:
     - pip
     - python
-    - wheel
-    - setuptools
+    - hatchling
   run:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - mockito
   requires:
     - pip
     - pytest
```

https://github.com/kaste/mockito-python/blob/1.5.4/pyproject.toml#L20
```suggestion
    - pytest >=4.6.11
```